src/2.0/sign.ts

```diff
@@ -13,10 +12,10 @@ export const signDocument = async <T extends OpenAttestationDocument>(
 ): Promise<SignedWrappedDocument<T>> => {
   const merkleRoot = `0x${document.signature.merkleRoot}`;
   const signature = await sign(algorithm, merkleRoot, keyOrSigner);
-  const proof = {
-    type: ProofType.OpenAttestationSignature2018,
+  const proof: Proof = {
+    type: "OpenAttestationSignature2018",
     created: new Date().toISOString(),
-    proofPurpose: ProofPurpose.AssertionMethod,
+    proofPurpose: "assertionMethod",
```

Comment:
this looks dodgy why is it a magic string here?

Reply:
ah fuck I forgot to use the literal

Reply:
ah actually it's fine I thought it was somewhere else. 

```
const proof: Proof = {
    type: "OpenAttestationSignature2018",
    created: new Date().toISOString(),
    proofPurpose: "assertionMethod",
    verificationMethod: SigningKey.guard(keyOrSigner)
      ? keyOrSigner.public
      : `did:ethr:${await keyOrSigner.getAddress()}#controller`,
    signature
  };
```

Proof declares which literal to use.

Reply:
ok